src/community/design-system-day/index.md

```diff
@@ -97,7 +97,7 @@ Most sessions are talks or show and tells where speakers may take questions at t
 
 #### In the morning
 
-- We’re feeling ‘appy
+- We’re feeling ’appy
```

Comment:
Text editors love flipping apostrophes the other way around 🤦‍♂️

Reply:
Thank you!